src/components/shared-components/PageSections/Juiciest/index.tsx

```diff
@@ -0,0 +1,74 @@
+import { Flex } from '@chakra-ui/react';
+import React from 'react';
+import { Link } from 'react-router';
+
+import { TEST_IDS } from '~/constants';
+import { PADDINGS } from '~/constants/styles';
+import { RecipeProps } from '~/types';
+
+import { ButtonViewMore, CategoryHeader } from '../..';
+import CategoryCard from '../../CategorySection/CategorySectionCard';
+
+interface JuiciestRecipesSectionProps {
+    recipes?: RecipeProps[];
+    markdownText?: string;
+}
+
+export const JuiciestRecipesSection: React.FC<JuiciestRecipesSectionProps> = ({
+    recipes,
+    markdownText,
+}) => {
+    if (!recipes?.length) return null;
+
+    return (
+        <Flex justifyContent='space-between' direction='column' mb={{ base: 8, xl: 10 }}>
+            <Flex justifyContent='space-between'>
+                <CategoryHeader title='Самое сочное' mb={PADDINGS.subsectionHeaderMb} />
+                <ButtonViewMore
+                    display={{ base: 'none', xl: 'flex' }}
+                    as={Link}
+                    to='/the-juiciest'
+                    data-test-id={TEST_IDS.juciestLink}
+                    title='Вся подборка'
+                    noButtonIcon={false}
+                />
+            </Flex>
+            <Flex flexWrap='wrap' gap={4}>
+                {recipes.map((card, index) => {
+                    const { title, description, category, image, id, likes, bookmarks } = card;

```

Comment:
можешь деструктуризировать прямо в момент объявления
```suggestion
                {recipes.map(({ title, description, category, image, id, likes, bookmarks }, index) => {

```

Reply:
Сделал